ResearchKit/ActiveTasks/ORKRangeOfMotionStep.h

```diff
@@ -0,0 +1,28 @@
+//
+//  ORKRangeOfMotionStep.h
+//  ResearchKit
+//
+//  Created by Darren Levy on 8/20/16.
+//  Copyright © 2016 researchkit.org. All rights reserved.
+//
+
+@import Foundation;
+#import <ResearchKit/ORKDefines.h>
+#import <ResearchKit/ORKActiveStep.h>
+#import <ResearchKit/ORKOrderedTask.h>
```

Comment:
Review the coding guidelines.
One thing here is to have 2 new lines before imports, and 2 new lines after imports.